src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/Resource.cs

```diff
@@ -0,0 +1,80 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
```

Comment:
Not sure what you mean here. I checked many other swaggers and they all include a definition of Resource. Are you saying that there is some special parameters to AutoRest so that resource.cs won't be generated for the SDK?